travis/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
+  if [ ! -z "$TRAVIS_TAG" ]; then
+    mvn clean deploy --settings travis/mvn-settings.xml -DskipTests=true -B -U -P oss-release "$@"
```

Comment:
can the skipTests go in the mvn-settings.xml so we got all the stuff in one place?